packages/vm/lib/evm/eei.ts

```diff
@@ -201,20 +200,18 @@ export default class EEI {
    * @param address - Address of account
    */
   async getExternalCodeSize(address: BN): Promise<BN> {
-    const addressBuf = addressToBuffer(address)
-    const code = await this._state.getContractCode(addressBuf)
+    const addr = new Address(addressToBuffer(address))
```

Comment:
Unrelated to this PR: `addressToBuffer` is defined in this file but should point to `ethereumjs-util` implementation of `addressToBuffer` (which seems to be the same)

Reply:
i'm not seeing an `addressToBuffer` in ethereumjs-util

Reply:
Never mind, thought this was in util due to https://github.com/ethereumjs/ethereumjs-util/issues/276